.cfnlintrc.yaml

```diff
@@ -4,5 +4,6 @@ ignore_checks:
   - E3001
   - W2001
   - W3045
+  - W8001
```

Comment:
ignores message `W8001 Condition DoCWReadOnly not used
templates/Cloudwatch/CrossAccountSharingRole-AccountList.yaml:29:3`